build.gradle

```diff
@@ -5,11 +5,12 @@ apply plugin: 'com.vanniktech.android.junit.jacoco'
 buildscript {
   ext.versions = [
     minSdk: 14,
-    compileSdk: 27,
+    compileSdk: 28,
     targetSdk: 27,
-    supportLibrary: '27.1.1',
-    espresso: '3.0.2',
-    testRules: '1.0.2',
+    appcompat: '1.0.0',
```

Comment:
Do we really need appcompat? Like I said I think only the annotations are being used?

Reply:
it's used in `/sample` -- `MainActivity extends AppCompatActivity`